Makefile

```diff
@@ -109,9 +109,16 @@ fmt:
 	@echo "===> Formatting Go files <==="
 	@gofmt -s -l -w $(GO_FILES)
 
+.PHONY: .linter
+.linter:
+	@if ! command golint; then \
+	  go get -u golang.org/x/lint/golint; \
+	fi
+
 .PHONY: lint
-lint:
-	golint $$(go list ./...)
+lint: export GOOS=linux
```

Comment:
I wonder what's the "best practice" way of doing this once we start to support multiple OSes.

Reply:
If we supported multi-OSes one day, I think we would not need the `GOOS`, there would be a implementation for the OS. For any exception, we can also put a `doc.go` maybe?

Reply:
Yes but ideally, a simple make target would be able to check the style of all files, including all OS-specific files. That's ok though, we only support Linux today so we can worry about it down the line.